### PR TITLE
Some more sign in/up & ichain stuff

### PR DESCRIPTION
**Pull request by hennevogel**

### CODE REVIEW

File: app/models/ability.rb [hunk 1]
```diff
@@ -49,12 +49,13 @@ def not_signed_in
...
-      registration.new_record?
-    end
-
-    can [:show, :create], Event do |event|
-      event.new_record?
+    unless CONFIG['authentication']['ichain']['enabled']
+      can [:show, :create], Registration do |registration|
+        registration.new_record?
+      end
+      can [:show, :create], Event do |event|
```
Comment (ChrisBr) [inline app/models/ability.rb]: It should be possible to show the event as unregistered user, shouldn't it?

File: config/environments/development.rb [hunk 2]
```diff
@@ -85,4 +85,12 @@
                               }
                             )
 
+
+  config.after_initialize do
```
Comment (ChrisBr) [inline config/environments/development.rb]: Why do we need this?

**Comment by ChrisBr** [comment]
and don't we need a conditional statement also here:
https://github.com/openSUSE/osem/blob/master/app/views/proposal/new.html.haml#L25

File: app/models/ability.rb (hunk 1 above)
Comment (hennevogel) [inline app/models/ability.rb]: [It is](https://github.com/hennevogel/osem/blob/master/app/models/ability.rb#L45) see above

File: config/environments/development.rb (hunk 2 above)
Comment (hennevogel) [inline config/environments/development.rb]: To make it easier to develop in ichain mode.

**Comment by hennevogel** [comment]
No as we never reach the case where there isn't a current user

File: app/models/ability.rb (hunk 1 above)
Comment (ChrisBr) [inline app/models/ability.rb]: :+1:

File: config/environments/development.rb (hunk 2 above)
Comment (ChrisBr) [inline config/environments/development.rb]: Okay, is it now enabled by default for development or how can I enable it?

**Comment by ChrisBr** [comment]
@hennevogel Ah yes true, because of the ability! Got it!

:ship: it!

File: config/environments/development.rb (hunk 2 above)
Comment (hennevogel) [inline config/environments/development.rb]: You still have to enable it in your [config](https://github.com/openSUSE/osem/blob/master/config/config.yml.example#L22) but when you do it is straight in test mode. No need to edit the [initializer](https://github.com/openSUSE/osem/blob/master/config/initializers/devise.rb#L270) anymore.

**Pull request merged**